src/transformations/segmentize.jl

```diff
@@ -169,12 +169,12 @@ This is useful for plotting geometries with a limited number of vertices, or for
 Returns a geometry of similar type to the input geometry, but resampled.
 """
 function segmentize(geom; max_distance, threaded::Union{Bool, BoolsAsTypes} = _False())
```

Comment:
```suggestion
@inline function segmentize(geom; max_distance, threaded::Union{Bool, BoolsAsTypes} = _False())
```

We should make sure to inline everything that has this bool/BoosAsTypes conversion so constprop works

Reply:
Will add a devdocs section in #108 about this as well then